### PR TITLE
[RDY] Merge NewickLoadService and GraphLoadService, add persistence.

### PR DESCRIPTION
**Pull request by gfokkema**
When persistence works, only the GraphProperty  should remain, see the FIXME's :)

Also add persistence.

### CODE REVIEW

**Comment by Unia** [comment]
Working nicely! Just the `GraphLoadServiceTest` must be extended, which @pvanagtmaal is working on atm.

**Comment by Unia** [comment]
Still working and looking fine! Next reviewer may merge!

File: dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java [hunk 1]
```diff
@@ -238,4 +245,29 @@ protected void analyze() {
...
+			ResourceIterator<Node> dbroots = e.execute(GET_PHYLO_ROOT).columnAs("n");
+			Node dbroot = dbroots.next();
+
+			return getTree(dbroot, new TreeNode(null));
+		});
+	}
+
+	private TreeNode getTree(Node parent, TreeNode treeparent) {
+		for (Relationship r : IteratorUtil.loop(parent.getRelationships(Direction.OUTGOING,
+										RelTypes.ANCESTOR_OF).iterator())) {
```
Comment (Balletie) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java]: `getRelationships` returns an `Iterable`, so there's no need to first get the `Iterator` through `iterator()`, and then turn it into an `Iterable` again with `IteratorUtil.loop`.

**Comment by Unia** [comment]
@Balletie's comment is fixed. Merging :)

**Pull request merged**